src/modules/tsilo/tsilo.c

```diff
@@ -250,15 +250,25 @@ static int fixup_ts_append(void** param, int param_no)
  */
 static int w_ts_append(struct sip_msg* _msg, char *_table, char *_ruri)
 {
-	str ruri = {0};
+	str tmp  = STR_NULL;
```

Comment:
Ok, I understand now, thanks :) 
Please do a clean PR and I think it can be merged.